docs/sources/mimir/references/configuration-parameters/index.md

```diff
@@ -1169,6 +1169,22 @@ instance_limits:
 # (experimental) When enabled only gRPC errors will be returned by the ingester.
 # CLI flag: -ingester.return-only-grpc-errors
 [return_only_grpc_errors: <boolean> | default = false]
+
+# (experimental) When enabled, only series currently owned by ingester according
+# to the ring are used when checking user per-tenant series limit.
+# CLI flag: -ingester.use-ingester-owned-series-for-limits
```

Comment:
I definitely plan to remove `ingester.track-ingester-owned-series` as soon as we test this feature in our production, and enable `-ingester.use-ingester-owned-series-for-limits` everywhere. Once `-ingester.use-ingester-owned-series-for-limits` is enabled by default, we could remove it too, although I'd keep it around for a release or two just in case we find some bugs, and need to disable the feature.